arcgis-ios-sdk-samples/Maps/Create and save a map/CreateSaveMapViewController.swift

```diff
@@ -52,8 +54,31 @@ class CreateSaveMapViewController: UIViewController, CreateOptionsViewController
         AGSAuthenticationManager.shared().oAuthConfigurations.add(config)
         AGSAuthenticationManager.shared().credentialCache.removeAllCredentials()
         
-        // Initially show the map creation UI.
-        performSegue(withIdentifier: "CreateNewSegue", sender: self)
+        // Temporarily unset the API key for this sample.
+        // Please see the additional information in the README.
+        let apiKey = AGSArcGISRuntimeEnvironment.apiKey
+        AGSArcGISRuntimeEnvironment.apiKey = ""
```

Comment:
It might be better to put these in the initializer, i.e. `required init?(coder aDecoder: NSCoder) {  // ... }` rather than in the `viewDidLoad`. See Phil's previous comment here: https://github.com/Esri/arcgis-runtime-samples-ios/pull/1067#discussion_r633618703.